### PR TITLE
Added note about limits on no of services and revisions

### PR DESCRIPTION
**Pull request by abrennan89**

### CODE REVIEW

**Comment by alanfx** [comment]
Looks good to me, @abrennan89! 
QE Approved

File: serverless/installing_serverless/installing-openshift-serverless.adoc [hunk 1]
```diff
@@ -22,24 +22,35 @@ The total size requirements to run {ServerlessProductName} are dependent on the
...
+
+* Maximum number of Knative services: 1000
+* Maximum number of Knative revisions: 1000
+====
+
+=== Additional requirements for advanced use-cases
+
+For more advanced use-cases such as logging or metering on {product-title}, you must deploy more resources. Recommended requirements for such use-cases are 24 CPUs and 96GB of memory.
+
+// If you have high availability (HA) enabled on your cluster, this requires between 0.5 - 1.5 cores and between 200m - 2GB of memory for each replica of the Knative Serving control plane.
```
Comment (bergerhoffer) [inline serverless/installing_serverless/installing-openshift-serverless.adoc:48]: @abrennan89 just checking if you needed these for anything before we merge?

**Review by bergerhoffer — COMMENTED** [review]
One quick question.

File: serverless/installing_serverless/installing-openshift-serverless.adoc (hunk 1 above)
Comment (abrennan89) [reply serverless/installing_serverless/installing-openshift-serverless.adoc:48]: Nope, not until tomorrow :) I have a PR to make everything like this live for GA when it's ready to go

**Pull request merged**